src/vaadin-form-layout.html

```diff
@@ -436,15 +448,18 @@
           }
         }
 
+        /** @private */
         _invokeUpdateStyles() {
           this.updateStyles();
         }
 
         /**
          * Set custom CSS property values and update the layout.
+         * @param {Object<string, string>=} properties
+         * @protected
          */
-        updateStyles(...args) {
-          super.updateStyles(...args);
+        updateStyles(properties) {
+          super.updateStyles(properties);
```

Comment:
This is overridden from PolymerElement.
I updated the arguments so it better matches the real API.
See https://polymer-library.polymer-project.org/3.0/api/polymer-element#PolymerElement-method-updateStyles
which says:
```
updateStyles(properties: Object=): void
```
but since it should always receive objects with string keys and values we can use a more exact type: `{Object<string, string>=}` which translates to this in TS defs ->
```ts
updateStyles(properties?: {[key: string]: string}): void;
```